lib/test.config.js

```diff
@@ -10,8 +10,6 @@ const config = require('bedrock').config;
 config.mongodb.name = 'bedrock_test';
 config.mongodb.host = 'localhost';
 config.mongodb.port = 27017;
```

Comment:
is this file actually used anywhere?  Very surprised to see `test.config.js` in the `lib` folder.  Seems like this may have been copied by accident at some point.

Reply:
It was a blissfull Febuary day in 2015. His hair was long and his Harley was ready. So a young @dlongley added the `test.config.js` file to this project. Would it cause confusion for other developers? Sure! But the road was waiting and he threw caution to the wind: https://github.com/digitalbazaar/bedrock-mongodb/commits/32487569ec0ef6ef119573d059576ee78161bf25/lib/test.config.js

Anyways, yes it is used to setup the test options. It looks like at somepoint we had separate bedrock events so libraries would require test options for `bedrock-test` and different presets for production. Removing this breaks the test suite in this library and every library that depends on it. Is this a spannard of a different age? Perhaps. I tried removing it and it broke everything. We can discuss removing it in a different PR.

Reply:
LOL!  

For future reference: https://github.com/digitalbazaar/bedrock-test/blob/master/lib/index.js#L41

bedrock-test in this age is responsible for loading the test config in the `test` folder in response to the bedrock event.

Reply:
if that is the case let's remove it (it looks like it is totally safe). Make an issue and once we get this out the door, I can remove that file, sym link my branch to all the dependent test projects, and make sure everything is still running smoothly. We do need to this release today so anything that could cause failures needs to be in the next revision.

Reply:
It wasn't a Harley, it was a Vespa.